Add Tibco EMS (#18029)
DataDog/integrations-core: tibco_ems/README.md

# Agent Check: tibco_ems

## Overview

> This check monitors [[TIBCO Enterprise Message Service][1] through the Datadog Agent. This integration allows you to do ...

Reviewer: Continue this suggestion please~

```suggestion
This check monitors [[TIBCO Enterprise Message Service][1] through the Datadog Agent. This integration allows you to 
```

# Agent Check: tibco_ems

## Overview

> This check monitors [tibco_ems][1] through the Datadog Agent.

Reviewer: Let's add some more information for users of this integration.

```suggestion
This check monitors [[TIBCO Enterprise Message Service][1] through the Datadog Agent. This integration allows you to do ...
```

## Setup

Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][3] for guidance on applying these instructions.

### Installation

> The tibco_ems check is included in the [Datadog Agent][2] package.

Reviewer: ```suggestion
The TIBCO EMS check is included in the [Datadog Agent][2] package.
```

No additional installation is needed on your server.

### Configuration

> 1. Edit the `tibco_ems.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your tibco_ems performance data. See the [sample tibco_ems.d/conf.yaml][4] for all available configuration options.

Reviewer: ```suggestion
1. Edit the `tibco_ems.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your TIBCO EMS performance data. See the [sample tibco_ems.d/conf.yaml][4] for all available configuration options.
```

2. [Restart the Agent][5].

### Validation

[Run the Agent's status subcommand][6] and look for `tibco_ems` under the Checks section.

## Data Collected

### Metrics

See [metadata.csv][7] for a list of metrics provided by this integration.

### Events

> The tibco_ems integration does not include any events.

Reviewer: ```suggestion
The TIBCO EMS integration does not include any events.
```

### Service Checks

> The tibco_ems integration does not include any service checks.
> 
> See [service_checks.json][8] for a list of service checks provided by this integration.

Reviewer: ```suggestion
See [service_checks.json][8] for a list of service checks provided by this integration.
```
